bigframes/operations/aggregations.py

```diff
@@ -228,21 +229,35 @@ def skips_nulls(self):
 
 
 class CutOp(WindowOp):
-    def __init__(self, bins: int):
-        self._bins_ibis = dtypes.literal_to_ibis_scalar(bins, force_dtype=Int64Dtype())
-        self._bins_int = bins
+    def __init__(self, bins: typing.Union[int, pd.IntervalIndex]):
+        if isinstance(bins, int):
+            self._bins_int = bins
```

Comment:
This might be a good place to check bins>=0. I know we are doing it in the cut method, but this is a public class as well.

Reply:
Done, added a check here.